Zend/zend_portability.h

```diff
@@ -336,10 +336,13 @@ char *alloca();
 # define ZEND_PRESERVE_NONE __attribute__((preserve_none))
 #endif
 
+
+#if !defined(__apple_build_version__) || (defined(__apple_build_version__) && __apple_build_version__ >= 17000404)
```

Comment:
Nit: please indent the nested cpp directives (with a space after the #)

Reply:
I do not get your suggestion @henderkes, it is not within a nested part is it ?